server/src/modules/users/domain/value-objects/errors.ts

```diff
@@ -6,14 +6,21 @@ export namespace UserValueObjectErrors {
     }
   }
 
-  export class InvalidPassword {
+  export class InvalidSecretValue {
```

Comment:
These `UserValueObjectErrors` don't actually extend `Error`, is this intentional?

Reply:
Hmm, we should be doing this to benefit from the error methods.